microsoft/testsuites/ltp/ltp.py

```diff
@@ -125,7 +125,7 @@ def run_test(
         self.run(parameters, sudo=True, force_run=True, timeout=12000)
 
         # to avoid no permission issue when copying back files
-        self.node.tools[Chmod].update_folder("/opt/ltp", "a+rwX", sudo=True)
+        self.node.tools[Chmod].update_folder("/opt", "a+rwX", sudo=True)
```

Comment:
@r-dailey Do you know why it is failing because I see that the read permission on ltp folder is already set?

Reply:
It is failing because the read permission on the /opt folder is set to root only and the test does not run the copyback operation as root.